packages/flutter/lib/src/painting/shader_warm_up.dart

```diff
@@ -101,12 +101,23 @@ class DefaultShaderWarmUp extends ShaderWarmUp {
     path.moveTo(60.0, 20.0);
     path.quadraticBezierTo(60.0, 60.0, 20.0, 60.0);
 
-    final List<ui.Path> paths = <ui.Path>[rrectPath, circlePath, path];
+    final ui.Path convexPath = ui.Path();
```

Comment:
drive by: I wonder how I will ever know if I can modify/remove this code?   A comment might be warranted.

Reply:
Good question. I've added additional comments/docs to address this question.

Writing (English) is not one of my strongest skills so it may still be suboptimal. But I'm very happy to learn and improve that skill. Please let me know if you have further suggestions 😄